README.md

```diff
@@ -39,19 +71,24 @@ configen --plist-path <plist> --hints-path <mapping-file> --class-name <output-c
 
 # e.g.
 
-configen --plist-path EnvironmentConfig/EnvironmentConfig.plist --hints-path EnvironmentConfig.map --class-name EnvironmentConfig --output-directory EnvironmentConfig
+configen --plist-path EnvironmentConfig/EnvironmentConfig_Prod.plist --hints-path EnvironmentConfig.map --class-name EnvironmentConfig --output-directory EnvironmentConfig
 
 ```
 
 `configen` generates Swift files by default. However, you can generate Objective-C files by providing `objc` as the final argument.
 
+The best way to support multiple environments is to define a separate scheme for each one. Then add the relevant target as an external build step for each scheme ensuring that 'Parallelize Build' is disabled.
+
+Please refer to the example project included in the repository for further guidance. 
+
 # Standard types supported
 
 * `Int`: Expects integer type in plist
 * `String`: Expects string type in plist
 * `Bool`: Expects Boolean type in plist
 * `Double`: Expects floating point type in plist
-* `NSURL`: Expects a string in the plist, which can be converted to an NSURL (validated at compile time)
+* `URL`: Expects a string in the plist, which can be converted to a URL (validated at compile time) 
```

Comment:
depending on what you think about my comment above, you might need to remove this reference to `NSURL` :)